### ISSUE TITLE
Test suite fails 

### ISSUE THREAD

**Issue by chrisknuteson-qumulex** [jeffijoe/typesync#37]
Greetings,

It looks like the automated tests fail when trying to hit the typedefsUrl. 
```»  TypeSync v0.5.1
⠧  Syncing type definitions in package.json...events.js:173
      throw er; // Unhandled 'error' event
      ^

Error: incorrect header check
    at Zlib.zlibOnError [as onerror] (zlib.js:162:17)
Emitted 'error' event at:
    at errorOrDestroy (internal/streams/destroy.js:98:12)
    at Gunzip.onerror (_stream_readable.js:717:7)
    at Gunzip.emit (events.js:197:13)
    at Zlib.zlibOnError [as onerror] (zlib.js:165:8)
error Command failed with exit code 1.
```

Simple reproduction case: Pull the repo; npm install; npm test

**Comment by jeffijoe** [comment]
Sounds like a transient error with the type definitions source. Try again?

**Comment by chrisknuteson-qumulex** [comment]
Maybe. I've had failing statuses for about 10 min now so I thought to run it across your bow

Is it possible some header change from the source is affecting the fetch? 
`https://typespublisher.blob.core.windows.net/typespublisher/data/search-index-min.json`

**Comment by jeffijoe** [comment]
The error is from the unzip. I'd imagine a new file is being uploaded and it's not complete yet? Maybe they stopped zipping it? Not sure, will have to wait and see.

**Comment by chrisknuteson-qumulex** [comment]
Thanks!

**Comment by jeffijoe** [comment]
Apparently unzipping is no longer needed. Shipped 0.6.1 to fix this.
